api-tests/features/mfa-reset-journey.feature

```diff
@@ -19,3 +20,56 @@ Feature: MFA reset journey
     Then I get an OAuth response
     When I use the OAuth response to get my MFA reset result
     Then I get a successful MFA reset result
+
+  Scenario: Successful MFA reset journey - with CI
+    When I submit an 'appTriage' event
+    Then I get a 'dcmaw' CRI response
+    When I submit 'kenneth-passport-needs-enhanced-verification' details to the CRI stub
+    Then I get a 'page-dcmaw-success' page response
+    When I submit a 'next' event
+    Then I get an OAuth response
+    When I use the OAuth response to get my MFA reset result
+    Then I get a successful MFA reset result
```

Comment:
Decided not to combine these with `Scenario Outline` because we want to run the successful mfa reset scenario in build but not the one with the CI